Day4/schema/create_raw_employee.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE raw_employee(
+    employee_id VARCHAR(500), 
+    first_name VARCHAR(500),
+    last_name VARCHAR(500), 
+    department_id VARCHAR(500), 
+    department_name VARCHAR(500), 
+    manager_employee_id VARCHAR(500), 
+    employee_role VARCHAR(500), 
+    salary VARCHAR(500), 
+    hire_date VARCHAR(500), 
+    terminated_date VARCHAR(500), 
+    terminated_reason VARCHAR(500), 
+    dob VARCHAR(500), 
+    fte VARCHAR(500), 
+    location VARCHAR(500)
+);
```

Comment:
Add a new line at the end @sanjeevbanmala

Reply:
I will add.

Reply:
```suggestion
);

```
I was expecting this, not a blank line before the last line.